modules/install-sno-requirements-for-installing-on-a-single-node.adoc

```diff
@@ -60,4 +60,19 @@ BMC is not supported on {ibm-z-name} and {ibm-power-name}.
 |Ingress route|`*.apps.<cluster_name>.<base_domain>`| Add a wildcard DNS A/AAAA or CNAME record that targets the node. This record must be resolvable by clients external to the cluster.
 |====
 +
+.Example output
+[source,terminal]
+----
+$ cluster_vip	api.$cluster_name.$base_dns_domain
+$ cluster_vip	oauth-openshift.apps.$cluster_name.$base_dns_domain
+$ cluster_vip	console-openshift-console.apps.$cluster_name.$base_dns_domain
+$ cluster_vip	grafana-openshift-monitoring.apps.$cluster_name.$base_dns_domain
+$ cluster_vip	thanos-querier-openshift-monitoring.apps.$cluster_name.$base_dns_domain
+$ cluster_vip	prometheus-k8s-openshift-monitoring.apps.$cluster_name.$base_dns_domain
+$ cluster_vip	alertmanager-main-openshift-monitoring.apps.$cluster_name.$base_dns_domain
```

Comment:
What is this format?

Reply:
I got this example output from @qJkee in [HCIDOCS-122](https://issues.redhat.com/browse/HCIDOCS-122).

Reply:
Not sure where they got that from, it's too verbose. We don't need to invent the wheel - surely the docs mention somewhere the required DNS records, can we copy from that / link to that?

Reply:
What about this? https://github.com/openshift/openshift-docs/blob/f513e56cd48cbd686cfbd47f93664faf96adab46/modules/installing-sno-requirements-for-installing-single-node-openshift.adoc#L36-L39

Reply:
Thanks for the feedback! If I'm understanding you correctly, you're suggesting that I revert my changes to this file (remove the "Example Output")? The [content](https://github.com/openshift/openshift-docs/blob/f513e56cd48cbd686cfbd47f93664faf96adab46/modules/installing-sno-requirements-for-installing-single-node-openshift.adoc?plain=1#L32-L41) you're suggesting that I use is already in the document. Thanks!